schema/build-config-schema.js

```diff
@@ -139,8 +139,13 @@ rules.forEach(function forRule(rule) {
           "type": "boolean",
           "default": true
         },
+        "headings": {
```

Comment:
Same comment about being sorted when generated. See npm scripts for command to do so.

Reply:
Oh, oops, this is not the generated file. Then please sort manually. :)